pyproject.toml

```
@@ -1,21 +1,40 @@
+[tool.poetry]
+name = "dynamic"
+version = "1.1"
+description = "A Modern, user-friendly command-line HTTP client for the API testing, and if you're stuck - Search and browse StackOverflow without leaving the CLI"
+authors = ["IndianOpenSourceFoundation"]
+license = "General Public License GNU"
+
```

Packages should be mentioned here
```suggestion
packages = [{ include = "dynamic" }]

```